src/js/controller/captions.js

```diff
@@ -108,14 +110,18 @@ const Captions = function(_model) {
         return list;
     }
 
-    function _selectDefaultIndex() {
+    function _selectDefaultIndex(selectAvailableTrack) {
         let captionsMenuIndex = 0;
         const label = _model.get('captionLabel');
 
         // Because there is no explicit track for "Off"
         //  it is the implied zeroth track
         if (label === 'Off') {
```

Comment:
`label === 'Off'` when the user elected to turn off captions. This should always take precedence. We should only turn on caption after this return statement if the ab test set an argument in the model that captions should be enabled AND we autostarted muted.

Reply:
If you call `localStorage.clear()` and refresh your test page, `_model.get('captionLabel')` will return `undefined`.